src/Acuminator/Acuminator.Tests/Tests/StaticAnalysis/DacDeclaration/DacForbiddenFieldsNonISVTests.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Acuminator.Analyzers.StaticAnalysis;
+using Acuminator.Analyzers.StaticAnalysis.DacDeclaration;
+using Acuminator.Analyzers.StaticAnalysis.EventHandlers;
+using Acuminator.Tests.Helpers;
+using Acuminator.Tests.Verification;
+using Acuminator.Utilities;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Xunit;
+
+namespace Acuminator.Tests.Tests.StaticAnalysis.DacDeclaration
+{
+	public class DacForbiddenFieldsNonISVTests : CodeFixVerifier
+	{
+
+		protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer() =>
+			new DacDeclarationAnalyzer(CodeAnalysisSettings.Default
+				.WithIsvSpecificAnalyzersDisabled());
```

Comment:
ISV is disabled by default

Reply:
Good, but it is not obvious for me. This line makes this setting obvious.

Reply:
Well why don't you also add WithStaticAnalysisEnabled. I'm not sure if it is obvious for you that the static analysis is enabled. 
I think we need to remember or quickly see the default values - it's not that hard. Only 4 of them
If we continue with this approach I can also demand that you change all static readonly strings to constant ones. Because IDE shows the values of constants to me in a tooltip but I can't see the value for a static readonly and it's not obvious to me what is stored there

Reply:
Ok, for now let's keep the settings as is